docs/notebooks/data/periwinkles.csv

```diff
@@ -0,0 +1,32 @@
+distance,direction
+107,1.1693705988362009
+46,1.1519173063162575
+33,1.2915436464758039
+67,1.064650843716541
+122,1.0122909661567112
+69,1.0471975511965976
+43,1.7453292519943295
+30,1.5533430342749532
+12,2.9845130209103035
+25,2.897246558310587
+37,1.710422666954443
+69,1.0471975511965976
+5,-2.8448866807507573
+83,1.710422666954443
+68,1.5009831567151235
+38,2.1467549799530254
+21,2.8797932657906435
+1,2.321287905152458
+71,1.7627825445142729
+60,1.8325957145940461
+71,1.239183768915974
+71,1.4660765716752369
+57,1.3089969389957472
+53,1.710422666954443
+38,1.4486232791552935
+70,1.239183768915974
+7,1.2915436464758039
+48,1.5882496193148399
+7,0.6632251157578453
+21,-2.7925268031909276
+27,0.9773843811168246
```

Comment:
Can we remove this file from the repo and add it to figshare? Do you want me to do it?